webpack.config.prod.js

```diff
@@ -16,11 +16,13 @@ const gitCommitSHA = () => {
 	}
 };
 
+const prefix = process.env.BUNDLE_PREFIX ?? '[chunkhash]/';
+
 module.exports = webpackMerge.smart(config, {
 	mode: 'production',
 	output: {
-		filename: `[chunkhash]/graun.standalone.commercial.js`,
-		chunkFilename: `[chunkhash]/graun.[name].commercial.js`,
+		filename: `${prefix}graun.standalone.commercial.js`,
```

Comment:
so that we can optionally build prod without a hash, so playwright can know file names to override the requests to.